validator/src/test/java/org/odftoolkit/odfvalidator/InvalidPackageTest.java

```diff
@@ -32,16 +32,22 @@
 /** Test some invalid packages. */
 public class InvalidPackageTest extends OdfValidatorTestBase {
 
-  @Test
-  public void validatePackage1() {
-    String output = "";
-    try {
-      String name = "testInvalidPkg1.odt";
-      output = doValidation(name, null);
-    } catch (Throwable t) {
-      t.printStackTrace();
-      Assert.fail(t.toString());
-    }
+    @Test
+    public void validatePackage1() {
+        String output = "";
+        try {
+            String name = "testInvalidPkg1.odt";
+            output = doValidation(name, null);
+        } catch (Throwable t) {
+            t.printStackTrace();
+            Assert.fail(t.toString());
+        }
+        System.out.println(output);
+        Assert.assertTrue(output.contains("testInvalidPkg1.odt/mimetype:  Error: The file 'mimetype' is not the first file in the ODF package"));
+        Assert.assertTrue(output.contains("testInvalidPkg1.odt/mimetype:  Error: The file 'mimetype' shall not be compressed"));
+        Assert.assertTrue(output.contains("testInvalidPkg1.odt/mimetype:  Error: There shall be no extra field for the 'mimetype' file"));
+        Assert.assertTrue(output.contains("testInvalidPkg1.odt/META-INF/manifest.xml:  Error: The file 'Configurations2/accelerator/current.xml' shall not be listed in the 'META-INF/manifest.xml' file as it does not exist in the ODF package"));
+        Assert.assertTrue(output.contains("testInvalidPkg1.odt:  Info: 8 errors, 10 warnings"));
```

Comment:
i dont get 8 errors here?
both in the 0.9 branch "master" and with this branch i get "7 errors".